website/redirects.next.js

```diff
@@ -34,6 +34,11 @@ module.exports = [
     destination: '/docs/auth/aws',
     permanent: true,
   },
+  {
+    source: '/docs/auth/jwt_oidc_providers',
+    destination: '/docs/auth/jwt/oidc_providers',
+    permanent: true,
+  },
```

Comment:
Needed this redirect to account for the moved file. Should note as well here: the reason this happened is that `docs-page` was previously not actually performing the "nesting structure" check - it now is, so these issues will be properly flagged and fail builds going forward.